### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by CusiniM**
Let's try to have as much granularity as possible and multiple people per folder. I have also started to remove myself and randy from a few of these so that we don't get asked to review basically every PR.  

I have broken down `inputFiles` and  `constitutive`. 

2 things to keep in mind:
 - deepest level rules
 - if we leave a folder blank it means no one owns it. I think this might be a good solution for some docs folders.
 

Go ahead and add yourselves wherever you see fit.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/multiphaseFlowFractures               @Cusinim @paveltomin 
```

File: .github/CODEOWNERS
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/materialPointMethod                @homel1 @cmcrook5 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/initialization                                 @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/multiphaseFlowFractures                   @paveltomin @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/proppant                                  @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/singlePhaseFlowFractures                  @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/solidMechanics                            @CusiniM @castelletto1 @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermalSinglePhaseFlowFractures           @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermoPoromechanics                       @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermoPoromechanicsFractures              @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/triaxialDriver                 @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/solidMechanics     @rrsettgast @castelletto1 @CusiniM @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan @jhuang2601
```

**Review by jhuang2601 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (ryar9534) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601 @ryar9534 
```

File: .github/CODEOWNERS
Comment (ryar9534) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan @ryar9534 
```

**Review by ryar9534 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS:14]: you should have a default for the subdirectories that don't have a listing

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/initialization                            @jhuang2601 @rrsettgast
```

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601 @matteofrigo5 
```

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/surfaceGeneration                         @CusiniM @castelletto1 @rrsettgast 
```

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin 
/src/coreComponents/constitutive/pvtPackage           @CusiniM @dkachuma @paveltomin @joshua-white
```

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/python             @cssherman @corbett5 @alexbenedicto 
```

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS:67]: Who is in charge of the vtk interface?

File: .github/CODEOWNERS
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/linearAlgebra                     @rrsettgast @castelletto1 @victorapm  @CusiniM 
```

**Review by rrsettgast — APPROVED** [review]
The problem with having a bunch of different folks as code owners for the various inputFiles subdirectories is that a sweeping change will be very hard to get approved.

@CusiniM Perhaps you and I should be on every code owner spec? I generally wait until others have done their reviews, so this doesn't have a "randy approved so it is good" effect.

File: .github/CODEOWNERS
Comment (CusiniM) [reply .github/CODEOWNERS:14]: this change was not intended.

**Comment by CusiniM** [comment]
> The problem with having a bunch of different folks as code owners for the various inputFiles subdirectories is that a sweeping change will be very hard to get approved.
> 

Yeah, I had not considered this. On the other hand, if a change affects all inputFiles maybe it's good to require a lot of codeowners coz it ensures that everyone's aware of it? 

> @CusiniM Perhaps you and I should be on every code owner spec? I generally wait until others have done their reviews, so this doesn't have a "randy approved so it is good" effect.

I was trying to reduce the amount of notifications we receive so that it's easier to identify what really needs attention. Maybe one of us is sufficient?

File: .github/CODEOWNERS
Comment (CusiniM) [reply .github/CODEOWNERS:14]: fixed.

File: .github/CODEOWNERS
Comment (CusiniM) [reply .github/CODEOWNERS:67]: I am not sure.

File: .github/CODEOWNERS
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/simplePDE                                 @rrsettgast @castelletto1 
```

File: .github/CODEOWNERS
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @rrsettgast @CusiniM @paveltomin @castelletto1 @frankfeifan @ryar9534 @jhuang2601
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS:53]:
```suggestion
/src/coreComponents/constitutive/docs                  @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/solid                @rrsettgast @CusiniM @paveltomin @castelletto1 @jhuang2601 
```

File: .github/CODEOWNERS
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/thermalConductivity  @CusiniM @dkachuma @paveltomin @jhuang2601 
```

**Review by jhuang2601 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (CusiniM) [reply .github/CODEOWNERS:53]: I would leave this blank so that it's excluded from the codeowners lists.

**Review by dkachuma — COMMENTED** [review]
Can we also add owners to `src/coreComponents/constitutiveDrivers`

**Comment by CusiniM** [comment]
> Can we also add owners to `src/coreComponents/constitutiveDrivers`

yes, of course.

File: .github/CODEOWNERS
Comment (MelReyCG) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/codingUtilities                   @rrsettgast @untereiner @corbett5 @wrtobin @MelReyCG
```

File: .github/CODEOWNERS
Comment (MelReyCG) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/dataRepository                    @rrsettgast @wrtobin @corbett5 @MelReyCG 
```

**Review by MelReyCG — COMMENTED** [review]
After discussing with @rrsettgast and @herve-gross, I would like to propose myself on the following folders:
```
/src/coreComponents/codingUtilities    
/src/coreComponents/dataRepository
```

**Review by rrsettgast — APPROVED** [review]

**Pull request merged**
